plugin-server/src/main/ingestion-queues/kafka-queue.ts

```diff
@@ -234,7 +137,7 @@ export class KafkaQueue implements Queue {
         }
     }
 
-    isPaused(targetTopic: string = this.pluginsServer.KAFKA_CONSUMPTION_TOPIC!, partition?: number): boolean {
+    isPaused(targetTopic: string = this.ingestionTopic, partition?: number): boolean {
```

Comment:
Does the default here work as you expect? We're referencing a dynamic variable at function definition time - I'd expect this to be undefined.

Reply:
Suggestion: Let's make the topic non-optional?

Reply:
This is a bit of a mess because currently both Kafka and Celery queues extend `Queue` and thus `Pausable`. I've spent some time here and there hacking around this but ultimately we just need #9865 to go in else I'll just be putting patches on top of that.

Reply:
Putting cleaning this up until yeet is in is :+1: in my book.